scapy/contrib/gtp_v2.py

```diff
@@ -390,6 +391,7 @@ class IE_Indication(gtp.IE_Base):
 
     ]
 
+
```

Comment:
Could you remove this unnecessary change ? Except this point, the PR looks good. Thanks!

Reply:
Sure, updated.